src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.jsx

```diff
@@ -21,7 +21,9 @@ export default class ConfigureIntegration extends AsyncView {
   }
 
   getTitle() {
-    return this.state.integration ? this.state.integration.name : 'Configure Integration';
+    return this.state.integration
+      ? this.state.integration.provider.name
```

Comment:
Should it be `t('%s Integration', this.state.integration.provider.name)` maybe?

Reply:
the full title is `Bitbucket - Sentry` so seems fine to me without